ED/src/dynamics/euler_driver.f90

```diff
@@ -9,6 +9,7 @@ subroutine euler_timestep(cgrid)
    use rk4_integ_utils
    use soil_respiration_module
    use photosyn_driv
+   use plant_hydro
```

Comment:
I know you weren't the one who started this, but as a general comment, use statements should always have the "only" list for clarity.